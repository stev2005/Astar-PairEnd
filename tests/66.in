TGCTGACGTGGAGGTACGGACCAGCAGCGCCGTGCCTGTGACGTACTCTGCATCTCGCGAGTGGCTTACTGGAACACACAGT
15
GAATG
CTTACCCCCGGAAAACCCGGGTACAGGACACGTAGCGAGTGATGC
TGAGATCCACTGCCTGTGGGTAGCTAGGGTGGCGAGCCTGGGATTTCTGTCCATTTTCG
AATCGTTAAGGGCGCAGGTAGACGTACGCTACT
TGTGGTCAGGGTGAGGAAGGGGGTAATACGAAATTGGAGAGCAACGGGGCA
TGTTGGAACAACTTACTGGCTGGCAATGCTGACCAAAAAC
AGTAGATCGCAATCATAGCT
C
GCCGC
TCCCACACCCTGC
CTAATTCTTTACACTGGGTTAACACGCAAGACCCCCATATGACG
AATTAGTATTCCACCGGGATACTTCGCCTTGAACTATGATCCACGGTAATTAAGTCCAGAAAAAC
ACCAAGCATTGTATGTTTGTATT
TTTATCTTGCTGTAG
ACG
