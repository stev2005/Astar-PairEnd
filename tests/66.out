1
20
27
14
26
19
8
0
1
5
23
33
11
5
0
